### PR TITLE
Modify the dependency management mode of generator to mod

### PR DESCRIPTION
**Pull request by Lionelpang**
delete the dep and use the go mod to manage the dependeces

### CODE REVIEW

**Comment by Lionelpang** [comment]
> a bit of context would be of help.
> 
> * Why?
> * Advantages?

When using the generator to update the model, I think it is too cumbersome to use dep to manage the dependency construction of golang, so I will change the secondary dependency management to the mod mode of golang, and then generate the generator without gopath configuration. Submit this modification. I hope to be able to merge this modification to simplify the way generator is built.

**Comment by rohanKanojia** [comment]
ok to test

**Comment by rohanKanojia** [comment]
I think this is a good change since go modules are now preferred way of dependency management in the go world.

Thanks a lot for this.

File: kubernetes-model/Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ build: gobuild
 	mvn clean install
 
 gobuild:
-	dep ensure -v
-	CGO_ENABLED=0 GO15VENDOREXPERIMENT=1 go build -a ./cmd/generate/generate.go
+	# dep ensure -v
```
Comment (rohanKanojia) [inline kubernetes-model/Makefile:25]: Could you please remove this comment?

Comment (Lionelpang) [reply kubernetes-model/Makefile:27]: Comments have been submitted

**Comment by rohanKanojia** [comment]
@Lionelpang : Could you please squash your commits so that we can proceed with merging your PR?

**Comment by Lionelpang** [comment]
> @Lionelpang : Could you please squash your commits so that we can proceed with merging your PR?

just add the changlog and remove the commen code。need to Launch a new PR？

**Comment by rohanKanojia** [comment]
@Lionelpang : No no, You don't need to lauch a new PR. You just need to squash your commits into one.

**Comment by rohanKanojia** [comment]
anyway, let me do just squash and merge instead. I want to rebase #2137 with this.

**Pull request merged**